lm_eval/__main__.py

```diff
@@ -103,21 +114,23 @@ def parse_eval_args() -> argparse.Namespace:
         "--include_path",
         type=str,
         default=None,
+        metavar="DIR",
         help="Additional path to include if there are external tasks to include.",
     )
     parser.add_argument(
         "--gen_kwargs",
-        default="",
+        default=None,
         help=(
             "String arguments for model generation on greedy_until tasks,"
-            " e.g. `temperature=0,top_k=0,top_p=0`"
+            " e.g. `temperature=0,top_k=0,top_p=0`."
         ),
     )
     parser.add_argument(
         "--verbosity",
         type=str,
         default="INFO",
-        help="Log error when tasks are not registered.",
+        metavar="INFO|DEBUG|WARNING|ERROR|CRITICAL",
+        help="Log debug error when tasks are not registered.",
```

Comment:
Change to `"Control the reported logging error level. Set to DEBUG when testing + adding new task configurations"` ?